packages/playground/blueprints/src/lib/steps/apply-wordpress-patches/wp-content/mu-plugins/0-playground.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Add a notice to wp-login.php offering the username and password.
+ */
+add_action(
+	'login_message',
+	function () {
+		return <<<EOT
+<div class="message info">
+	<strong>username:</strong> <code>admin</code><br /><strong>password</strong>: <code>password</code>
+</div>
+EOT;
+	}
+);
+
+/**
+ * Because the in-browser Playground doesn't have access to the internet, 
+ * network-dependent features like directories don't work. Normally, you'll 
+ * see a confusing message like "An unexpected error occurred." This mu-plugin
+ * makes it more clear that the feature is not yet supported.
+ * 
+ * https://github.com/WordPress/wordpress-playground/issues/498
+ */
+add_filter('plugins_api_result', function ($res) {
+	if ($res instanceof WP_Error) {
+		$res = new WP_Error(
+			'plugins_api_failed',
+			'Enable networking support in Playground settings to access the Plugins directory. Network access is an <a href="https://github.com/WordPress/wordpress-playground/issues/85">experimental, opt-in feature</a>. If you don\'t want to use it, you can still upload plugins or install them using the <a href="https://wordpress.github.io/wordpress-playground/query-api">Query API</a> (e.g. ?plugin=coblocks).'
```

Comment:
❤️ love this error message.

Reply:
Thanks! I wanted to make sure the next steps are crystal clear